sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -219,10 +222,26 @@ export class AssessmentController {
   ): Promise<MustReturnJobActionAcknowledgement> {
     const jobLogger = new Logger(job.type);
     try {
+      const assessment =
+        await this.studentAssessmentService.getAssessmentSummary(
+          job.variables.assessmentId,
+        );
+      if (!assessment) {
```

Comment:
i believe at this point the assessment cannot be empty and this check will be obsolete. To fetch the studentId and the programYearId i believe this call can happen after the updateAssessmentStatusAndSaveWorkflowData call.

Reply:
This is to validate the assessment id @guru-aot . We do the same in other places .e.g `loadAssessmentConsolidatedData`

Reply:
Yes i get it but this place we will not have a scenario where we will not have an assessment id.